client/src/components/form/input-component.tsx

```diff
@@ -62,6 +68,34 @@ const InputComponent = ({
 }) => {
   const { watch, register } = form;
   const { name, onChange, value } = field;
+  if (type === 'wysiwyg') {
+    const watchField = watch(name) as string;
+    const counterId = `${name} - counter`;
+    const hasError: boolean = !!watchField && !!maxSize && watchField.length > maxSize;
+    return (
+      <div className="w-full">
+        <ReactQuill
```

Comment:
One final comment: when the user writes content on a new line (in the wysiwyg), there is no visual space between that new one and the previous one, whereas when we render the content in the organisation/project detail page, we do have some space between lines. We don't want users entering multiple new lines in the form because that would cause excessive space in the detail page.

Could we visually have more space between paragraphs in the form? If possible, we'd like to have the same styles we have on the detail page for the lists as well.

Reply:
I made all this changes except for the required option. We are not using it right now and it will demand some complex changes. So I think is better to wait to see if this user case arises. Let me know if you think differently